src/manifolds/PowerManifold.jl

```diff
@@ -839,6 +839,21 @@ function vector_bundle_transport(fiber::VectorSpaceType, M::PowerManifold)
     return PowerVectorTransport(ParallelTransport())
 end
 
+function vector_transport_direction(M::AbstractPowerManifold, p, X, d)
+    return vector_transport_direction(M, p, X, d, PowerVectorTransport(ParallelTransport()))
+end
+
+function vector_transport_direction!(M::AbstractPowerManifold, Y, p, X, d)
```

Comment:
The `PowerVectorTransport` was used to avoid ambiguities, right?

Reply:
Yes, exactly, there were many ambiguities otherwise.

Reply:
Ok.